components/identity-event/org.wso2.carbon.identity.event/src/main/java/org/wso2/carbon/identity/event/IdentityEventConstants.java

```diff
@@ -72,8 +72,24 @@ private Event(){}
         public static final String POST_DELETE_USER_CLAIM= "POST_DELETE_USER_CLAIM";
         public static final String PRE_ADD_ROLE= "PRE_ADD_ROLE";
         public static final String POST_ADD_ROLE= "POST_ADD_ROLE";
+        public static final String PRE_GET_ROLES = "PRE_GET_ROLES";
+        public static final String POST_GET_ROLES = "POST_GET_ROLES";
+        public static final String PRE_GET_ROLE = "PRE_GET_ROLE";
+        public static final String POST_GET_ROLE = "POST_GET_ROLE";
+        public static final String PRE_UPDATE_ROLE_NAME = "PRE_UPDATE_ROLE_NAME";
+        public static final String POST_UPDATE_ROLE_NAME = "POST_UPDATE_ROLE_NAME";
         public static final String PRE_DELETE_ROLE= "PRE_DELETE_ROLE";
         public static final String POST_DELETE_ROLE= "POST_DELETE_ROLE";
+        public static final String PRE_GET_USER_LIST_OF_ROLE = "PRE_GET_USER_LIST_OF_ROLE";
+        public static final String POST_GET_USER_LIST_OF_ROLE = "POST_GET_USER_LIST_OF_ROLE";
+        public static final String PRE_GET_GROUP_LIST_OF_ROLES = "PRE_GET_GROUP_LIST_OF_ROLES";
+        public static final String POST_GET_GROUP_LIST_OF_ROLES = "POST_GET_GROUP_LIST_OF_ROLES";
+        public static final String PRE_UPDATE_GROUP_LIST_OF_ROLE = "PRE_UPDATE_GROUP_LIST_OF_ROLE";
+        public static final String POST_UPDATE_GROUP_LIST_OF_ROLE = "UPDATE_GROUP_LIST_OF_ROLE";
```

Comment:
shouldn't the value of this be POST_UPDATE_GROUP_LIST_OF_ROLE ?

Reply:
Addressed with commit 24f0165df0577d035bb0473ae2094d5a70751997